website/docs/cdktf/concepts/resources.mdx

```diff
@@ -370,6 +370,78 @@ import imports.kubernetes.namespace.NamespaceMetadata;
 
 </CodeTabs>
 
+## Refactoring & Renaming Resources
+
+When working with your infrastructure definitions and the need arises to refactor or rename resources without destroying and recreating them, you can leverage the `moveTo` function. This section outlines how to perform resource moves, rename resources, and enable foreach on existing resources in your CDKTF project.
+
+### Performing Resource Moves
+
+The `moveTo` function is available on all resources and is used for relocating a resource to the location specified by the string target `moveTarget`. To set the `moveTarget` for a resource, utilize the `addMoveTarget` function that is present on the resource to move to.
```

Comment:
```suggestion
The `moveTo` function is available on all resources and is used for relocating a resource to the location specified by the string target `moveTarget`. To set a resource's `moveTarget`, use the `addMoveTarget` function that is present on the resource to move to. You can specify an arbitrary string for the `moveTarget`, but it must be unique within your stack.
```
I'm having trouble parsing "use the `addMoveTarget` function that is present on the resource to move to". It might be helpful to use a more concrete verb to describe what you do with the `addMoveTarget` function, for example "add the `addMoveTarget` function to the destination resource. Terraform moves the resource configured with the `moveTo` function to the target during the `apply` operation." I just made that up, of course, but it I think it would be helpful to include that level of (accurate) detail.

Reply:
Huge improvement on that wording, [included 👍].(https://github.com/hashicorp/terraform-cdk/pull/3152/commits/70e4146b79b73f51e3b568df7fd5918366cb5b49)